benchmarks/htap/lib/stats.py

```diff
@@ -14,7 +14,34 @@
 register_uuid()
 
 QUERY_TYPES = ['new_order', 'payment', 'order_status', 'delivery', 'stock_level']
-HISTORY_LENGTH = 10
+
+class OLTPBucketStats:
+    def __init__(self):
+        self.ok_transactions = 0
+        self.err_transactions = 0
+        self.min_runtime = float('inf')
+        self.max_runtime = 0
+        self.acc_runtime = 0
+
+    def add_sample(self, status, runtime):
+        if status == 'ok':
+            self.ok_transactions += 1
+        else:
+            self.err_transactions += 1
+
+        runtime = math.ceil(runtime * 1000)
```

Comment:
why the ceil if you then store a float into min_runtime and max_runtime? (judging from line 22)

Reply:
removed